static/sass/_pattern_cve.scss

```diff
@@ -145,10 +145,6 @@
       }
 
       .p-accordion__tab {
-        z-index: -1;
```

Comment:
This fixes the side nav categories not toggling when clicked eg:
[Screencast from 09.09.2024 17:01:50.webm](https://github.com/user-attachments/assets/16636a8d-20ce-4180-975c-9d3571dbcf00)